org.gimp.GIMP.json

```diff
@@ -417,48 +417,36 @@
             ]
         },
         {
-            "name": "babl",
+            "name": "cairo",
             "buildsystem": "meson",
-            "config-opts": [ "-Dwith-docs=false" ],
             "sources": [
                 {
-                    "type": "git",
-                    "url": "https://gitlab.gnome.org/GNOME/babl.git",
-                    "tag": "BABL_0_1_84",
-                    "commit": "ff6ac9e6a92a16ae5c5e233432c2fcaa22cb1c5e"
+                    "type": "archive",
+                    "url": "https://www.cairographics.org/releases/cairo-1.16.0.tar.xz",
+                    "sha256": "5e7b29b3f113ef870d1e3ecf8adf21f923396401604bda16d44be45e66052331"
+                },
+                {
+                    "type": "file",
```

Comment:
make it "type": "patch" and remove the section below and it should work.

Reply:
Oh really? Flatpak docs doesn't say that type "patch" can have a "url" instead of "path". Let's try. 🙂

Reply:
if it doesn't they it should be added to the repository. That's what I would have done in the first place.

Reply:
tbf I thought it would work.

Reply:
Well doesn't work:

> (flatpak-builder:3615397): flatpak-builder-WARNING **: 16:33:02.884: Unknown property url for type BuilderSourcePatch

Trying now with "path" but leaving an URL in there.

Reply:
As I said, adding the patch to the repository is likely the only way.

Reply:
> if it doesn't they it should be added to the repository. That's what I would have done in the first place.

Yeah that's what I will do if my last test fails again. I wanted to avoid duplication.
Yet this is driving me a bit nuts 🤪 and such a time waste.